examples/README.md

```diff
@@ -2,17 +2,21 @@
 
 ⭐ = ["Tier 1"](#tier-1)
 
+🎩 = [RHEL compatible](#rhel-compatible)
```

Comment:
I'm not sure we need this emoji

Reply:
Ok, but there are multiple identical entries